src/remediations/controller.fifi.js

```diff
@@ -45,7 +53,18 @@ exports.executePlaybookRuns = errors.async(async function (req, res) {
         throw new errors.Forbidden();
     }
 
-    const status = await fifi.getConnectionStatus(remediation, req.identity.account_number, req.entitlements.smart_management);
+    const rhcEnabled = await fifi.checkRhcEnabled();
+
+    if (!rhcEnabled && !req.entitlements.smartManagement) {
+        throw new errors.Forbidden();
+    }
```

Comment:
The same changes you made to `connection_status` are also applicable here.

Reply:
@jharting mirrored changes from connection_status in "execute" logic.